src/main/java/org/motechproject/prevac/service/impl/VisitRescheduleServiceImpl.java

```diff
@@ -146,14 +149,6 @@ private void checkNumberOfPatients(VisitRescheduleDto dto, Clinic clinic) { //NO
     }
 
     private void validateDate(VisitRescheduleDto dto, Visit visit) {
-        if (visit.getDate() != null) {
```

Comment:
I think it should work like this:
- if actual date is not null it should be validated (no planned date validation)
- if actual date is null, planned date is validated 
- I think we can use "IgnoreDateLimitation" for both

Reply:
Like I wrote in other comment, I am wondering about IgnoreDateLimitation for Actual Date.